src/api/statements/JSONLDObject/index.js

```diff
@@ -25,7 +25,7 @@ const merge = (field, fields, currentOutput, data) => {
     }
 
     if (completesAnotherField) {
-        return mergeCompleteField(currentOutput, field, fields, data);
+        return mergeCompleteField(currentOutput, field, fields, data); // FIXME: mergeCompleteField is async!!! JSONLDObject can also be async
```

Comment:
nohing to fix here.
`return` and `await` inside an async function are smart. They resolve the value they receive if it is a promise. Otherwise they leave tha value as is.